roboflow/models/video.py

```diff
@@ -65,7 +65,7 @@ def __init__(
         """  # noqa: E501 // docs
         self.__api_key = api_key
 
-    def predict(  # type: ignore[override]
+    def predict(
```

Comment:
This is an example where mypy do better. This override with incompatible signature is bad.

I don't know how to fix it without breaking compatibility, but I like to have it annotated as an issue.